antonpalinau/nodejs-aws-be
# added bff-service

3 - A working and correct express application should be in the bff-service folder. Reviewer can start this application locally with any valid configuration in the .env file and this application should works as described in the task 9.1
5 - The bff-service should be deployed with Elastic Beanstalk. The bff-service call should be redirected to the appropriate service : product-service or CART. The response from the bff-service should be the same as if product-service or CART services were called directly.

+1 - Add a cache at the bff-service level for a request to the getProductsList function of the product-service. The cache should expire in 2 minutes.
How to test:
Get products list
Create new product
Get products list - result shouldn’t have new product
Wait more than 2 minutes
Get products list - result should have new product
+1 - Use NestJS to create bff-service instead of express

http://antonpalinau-bff-api-dev.eu-west-1.elasticbeanstalk.com/products
http://antonpalinau-bff-api-dev.eu-west-1.elasticbeanstalk.com/cart/profile/cart - the same kind of request goes from fe app
